supervised/base_automl.py

```diff
@@ -1504,16 +1524,137 @@ def _base_predict(self, X, model=None):
         else:
             return predictions
 
-    def _predict(self, X):
-        predictions = self._base_predict(X)
-        # Return predictions
-        # If classification task the result is in column 'label'
-        # If regression task the result is in column 'prediction'
-        return (
-            predictions["label"].to_numpy()
-            if self._ml_task != REGRESSION
-            else predictions["prediction"].to_numpy()
-        )
+    def _predict(self, X, prediction_mode='best', n_models=1, custom_models=[]):
+        """
+        Generates predictions using one or multiple models based on the selected prediction mode.
+
+        Parameters
+        ----------
+        X : array-like, pandas.DataFrame
+            Input data to generate predictions for.
+
+        prediction_mode : str, default='best'
```

Comment:
Thank you for PR! You assumed that user already has a list with loaded models. Do you have example code for such use?